src/provider/rpc.ts

```diff
@@ -29,27 +30,30 @@ import {
   getSimulateTransactionOptions,
   waitForTransactionOptions,
 } from '../types';
-import { getAbiContractVersion } from '../utils/calldata/cairo';
-import { isSierra } from '../utils/contract';
-import { RPCResponseParser } from '../utils/responseParser/rpc';
-import { GetTransactionReceiptResponse, ReceiptTx } from '../utils/transactionReceipt';
 import type { TransactionWithHash } from '../types/provider/spec';
 import assert from '../utils/assert';
-import { hexToBytes, toHex } from '../utils/num';
+import { getAbiContractVersion } from '../utils/calldata/cairo';
+import { isSierra } from '../utils/contract';
 import { addHexPrefix, removeHexPrefix } from '../utils/encode';
+import { hexToBytes, toHex } from '../utils/num';
 import { wait } from '../utils/provider';
+import { RPCResponseParser } from '../utils/responseParser/rpc';
+import { GetTransactionReceiptResponse, ReceiptTx } from '../utils/transactionReceipt';
 import { LibraryError } from './errors';
 import { ProviderInterface } from './interface';
 
 export class RpcProvider implements ProviderInterface {
-  private responseParser: RPCResponseParser;
+  public responseParser: RPCResponseParser;
```

Comment:
I would suggest making this `private` again or adding an export for the `RPCResponseParser` class somewhere so it is exposed by the library and can be picked up by the automated documentation generation.

Reply:
c2887ed250c0f7331986738f6d3ef563eb0ea0ce